packages/edit-post/src/components/header/fullscreen-mode-close/index.js

```diff
@@ -18,21 +18,24 @@ import { wordpress } from '@wordpress/icons';
 import { store as editorStore } from '@wordpress/editor';
 import { store as coreStore } from '@wordpress/core-data';
 import { useReducedMotion } from '@wordpress/compose';
+import { useCallback } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import { store as editPostStore } from '../../../store';
 
 function FullscreenModeClose( { showTooltip, icon, href } ) {
-	const { isActive, isRequestingSiteIcon, postType, siteIconUrl } = useSelect(
-		( select ) => {
-			const { getCurrentPostType } = select( editorStore );
+	const { isActive, isRequestingSiteIcon, postType, siteIconUrl, goBack } =
+		useSelect( ( select ) => {
+			const { getCurrentPostType, getEditorSettings } =
+				select( editorStore );
 			const { isFeatureActive } = select( editPostStore );
 			const { getEntityRecord, getPostType, isResolving } =
 				select( coreStore );
 			const siteData =
 				getEntityRecord( 'root', '__unstableBase', undefined ) || {};
+			const _goBack = getEditorSettings()?.goBack;
```

Comment:
Looks like the unit tests are failing for this one (error calling `getEditorSettings()` 🤔:

```
FullscreenModeClose › when in full screen mode › should display a default site icon if no user uploaded site icon exists

    expect(jest.fn()).not.toHaveErrored(expected)

    Expected mock function not to be called but it was called with:
    [[Error: Uncaught [TypeError: getEditorSettings is not a function]]],[[Error: Uncaught [TypeError: getEditorSettings is not a function]]],["The above error occurred in the <FullscreenModeClose> component:·
```

Reply:
Yep, already on it 👍🏻  Thanks!